editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -142,9 +142,11 @@ void HelpBrowser::createActions() {
     connect(ovrAction, SIGNAL(triggered()), this, SLOT(resetZoom()));
     ovrAction->addToWidget(this);
 
+    // eval actions are added to mWebView->focusProxy() in onPageLoad()
     mActions[Evaluate] = ovrAction = new OverridingAction(tr("Evaluate as Code"), this);
     connect(ovrAction, SIGNAL(triggered()), this, SLOT(evaluateSelection()));
-    ovrAction->addToWidget(this);
+    mActions[EvaluateRegion] = ovrAction = new OverridingAction(tr("Evaluate as Code Region"), this);
+    connect(mActions[EvaluateRegion], &OverridingAction::triggered, this, [=]() { this->evaluateSelection(true); });
```

Comment:
```suggestion
    mActions[EvaluateRegion] = new OverridingAction(tr("Evaluate as Code Region"), this);
    connect(mActions[EvaluateRegion], &OverridingAction::triggered, this, [this]() { this->evaluateSelection(true); });
```

since you don't use ovrAction